tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1506,36 +1506,139 @@ Status Converter::RenameAndMarkOutputTensors(
 }
 
 #if IS_TRT_VERSION_GE(7, 1, 3, 0)
+enum class Tactic : int64_t { kINVALID_TACTIC = int64_t(0xD15EA5EDD15EA5ED) };
+
+enum class LayerImpl : int64_t {
+  kSHUFFLE = 16,
+};
+
 // An algorithm selector that always returns a specific ID for selectAlgorithms.
 // This is used to support the implementation of using environment variable
 // `TF_TRT_FIXED_ALGORITHM_ID` for debugging TensorRT.
```

Comment:
Let's fix this comment:
An algorithm selector to support debugging and work around known TensorRT issues.

Reply:
Fixed